.gitlab-ci.yml

```diff
@@ -100,15 +100,14 @@ test-rel-lin: #  most comprehensive tests, force all suggests, also integration
     _R_CHECK_FORCE_SUGGESTS_: "TRUE"
     OPENBLAS_MAIN_FREE: "1"
     TEST_DATA_TABLE_WITH_OTHER_PACKAGES: "TRUE"
-  before_script:
-    - mkdir -p ~/.R
-    - echo 'CFLAGS=-g -O3 -flto -fopenmp -Wall -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' > ~/.R/Makevars
-    - echo 'CXXFLAGS=-g -O3 -flto -fopenmp -Wall -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' >> ~/.R/Makevars
   script:
     - Rscript -e 'source("ci.R"); install.packages(dcf.dependencies(c("DESCRIPTION","inst/tests/tests-DESCRIPTION"), which="all"))'
     - *copy-src
     - rm -r bus
     - *move-src
+    - mkdir -p ~/.R
+    - echo 'CFLAGS=-g -O3 -flto -fopenmp -Wall -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' > ~/.R/Makevars
+    - echo 'CXXFLAGS=-g -O3 -flto -fopenmp -Wall -pedantic -fstack-protector-strong -D_FORTIFY_SOURCE=2' >> ~/.R/Makevars
```

Comment:
this is the fix for current failure due to `-flto` applied to jsonlite (and others) that fail when compiling with this flag

Reply:
Is this something we should file an issue with for those pkg?

Reply:
They should already know via their CRAN check pages, like data.table shows the red LTO fail currently.

Reply:
OK, in fact yes:

![Screenshot 2019-09-17 at 2 15 01 PM](https://user-images.githubusercontent.com/7606389/65015944-906cae80-d955-11e9-9d54-06e08833bf86.png)

(`jsonlite` here)